packages/platform/atoms/gcal-connect/GcalConnect.tsx

```diff
@@ -0,0 +1,37 @@
+import { cn } from "@/lib/utils";
+import type { FC } from "react";
+
+import { Button } from "@calcom/ui";
+import { CalendarDays } from "@calcom/ui/components/icon";
+
+import { useAtomsContext } from "../hooks/useAtomsContext";
+import { useGcal } from "../hooks/useGcal";
+
+interface GcalConnectProps {
```

Comment:
I think maybe it would be nice to add another prop - "alreadyConnectedClassName", so that connected Gcal stands out more instead of having the same style.